jms/1.7/modules/ROOT/pages/jms-studio-configuration.adoc

```diff
@@ -0,0 +1,473 @@
++
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>org.apache.activemq</groupId>
+    <artifactId>activemq-broker</artifactId>
+    <version>5.14.4</version>
+</dependency>
+----
+
+* *ActiveMQ Client* +
+The `activemq-client` of your choice that provides a valid `org.apache.activemq.ActiveMQConnectionFactory` implementation.
++
+WARNING: This dependency is always required.
++
+
+In your `pom.xml` file in Studio, adding the dependency looks as follows:
++
+[source,xml,linenums]
+----
+<dependency>
+     <groupId>org.apache.activemq</groupId>
+     <artifactId>activemq-client</artifactId>
+     <version>5.14.4</version>
+ </dependency>
+----
+
+To configure these libraries in the global element view follow these steps:
+
+. In Studio, navigate to the *Global Elements* tab.
+. Click *Create*.
+. In the filter box type `jms`, and select *JMS Config*.
+. Click *OK*.
+. In the *Connection* field select *ActiveMQ Connection*.
+. In the *Required Libraries* section, that shows the *ActiveMQ KahaDB*, *ActiveMQ Broker* and *ActiveMQ Client* libraries, click the *Configure...* button to install the dependency.
+. Select any of the install options:
+* *Add recommended library* +
+Installs the recommended library.
+* *Use local file* +
+Browse a local file for the required engine library and install it.
+* *Add Maven dependency* +
+Browse the dependency and install it.
+
+
+==== Generic External Libraries
+
+For a *Generic Connection*, provide all the libraries that your connection factory of your implementation requires. Be careful of adding all the dependencies to the application.
+
+In the following example, you configure a generic library for a connection that uses JMS 2.0 specification. Thereby, you need a generic connection with a different client library, like Apache Artemis:
+
+In your `pom.xml` file in Studio, adding the dependency looks as follows:
+
+[source,xml,linenums]
+----
+<dependency>
+    <groupId>org.apache.activemq</groupId>
+    <artifactId>artemis-jms-client-all</artifactId>
+    <version>2.17.0</version>
+</dependency>
+----
+
+To configure these libraries in the global element view follow these steps:
+
+. In Studio, navigate to the *Global Elements* tab.
+. Click *Create*.
+. In the filter box type `jms`, and select *JMS Config*.
+. Click *OK*.
+. In the *Connection* field select *Generic Connection*.
+. In the *Required Libraries* section, that shows the *JMS Client* click the *Configure...* button to install the dependency.
+. Select any of the install options:
+* *Add recommended library* +
+Installs the recommended library.
+* *Use local file* +
+Browse a local file for the required engine library and install it.
+* *Add Maven dependency* +
+Browse the dependency and install it.
+
+=== Configure the JMS Specification
+
+The JMS *Specification* you configure must be supported by the broker implementation associated to the connection, otherwise the connection fails.
```

Comment:
```suggestion
The JMS *Specification* you configure must be supported by the broker implementation associated with the connection, otherwise the connection fails.
```